ai2thor/tests/test_unity.py

```diff
@@ -414,7 +432,6 @@ def test_action_dispatch_find_conflicts_stochastic(controller):
     known_conflicts = {
         'GetComponent': ['type'],
         'StopCoroutine': ['routine'],
-        'ProcessControlCommand': ['controlCommand'],
```

Comment:
This is the one thing I'd like to specifically call out. I don't know why this was here before, since AgentManager's `ProcessControlCommand` never had a parameter named `controlCommand` (only `msg`: https://github.com/allenai/ai2thor/blob/master/unity/Assets/Scripts/AgentManager.cs#L1055).

With it, the tests fail, and removing it passes the tests.

Reply:
This is where the conflict would have arisen from:
https://github.com/allenai/ai2thor/blob/master/unity/Assets/Scripts/BaseFPSAgentController.cs#L1453

If the test isn't finding that then something is wrong with the PR.

Reply:
i just scanned through the changes - it looks like controlCommand was changed to serverAction in your PR, so it might be fine to remove.